packages/react-tooltip/src/index.ts

```diff
@@ -1 +1,2 @@
-import './version';
+// TODO: replace with real exports
+export {};
```

Comment:
FYI @behowell I had to add this after removing the version file import to work around an API Extractor error. So be sure to remove it once you start working on the real code.

Reply:
Will do, thanks for the heads up.